docs/content/posts/2023-03-09-release-1.17.0.md

```diff
@@ -0,0 +1,481 @@
+
+Please see [FLINK-30836](https://issues.apache.org/jira/browse/FLINK-30836) for
+more details.
+
+This release also widens the scope of sharing memory between slots to
+TaskManager, which can help to increase the memory efficiency if the memory
+usage of slots in a TaskManager is uneven. Furthermore, it can reduce the
+overall memory consumption at the expense of resource isolation after tuning.
+Read more about
+[tate-backend-rocksdb-memory-fixed-per-tm](https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/deployment/config/#state-backend-rocksdb-memory-fixed-per-tm)
+configuration. 
+
+
+## Calcite Upgrade
+
+Flink 1.17 is upgraded to [Calcite](https://calcite.apache.org/) version 1.29.0
+to improve the performance and efficiency of the Flink SQL system. Flink 1.16
+uses Calcite 1.26.0 which has severe issues with RexNode simplification caused
+by the SEARCH operator. This leads to wrong data from query optimization as
+reported in[CALCITE-4325](https://issues.apache.org/jira/browse/CALCITE-4325),[
+CALCITE-4352](https://issues.apache.org/jira/browse/CALCITE-4352). By upgrading
+to the latest version of Calcite, Flink can take advantage of its improved
+performance and new features in Flink SQL processing. This resolves multiple
+bugs and leads to faster query processing times. The downside of this upgrade is
+that the execution plan changes for certain SQL queries. Savepoints are not
+backwards compatible in these cases. 
+
+
+# Others
+
+
+## PyFlink
+
+The Flink 1.17 release includes updates to PyFlink, the Python interface for
+Apache Flink. Notable improvements include support for Python 3.10 and execution
+capabilities on Apple Silicon chipsets, such as the Mac M1 and M2 computers.
+Additionally, the release includes minor optimizations that enhance
+cross-process communication stability between Java and Python processes, enable
+the specification of data types of Python UDFs via strings to improve usability,
+and support access to job parameters in Python UDFs. This release focuses on
+improving PyFlink's functionality and usability, rather than introducing new
+major features. However, these enhancements are expected to improve the user
+experience and facilitate efficient data processing.
+
+
+## Daily Performance Benchmark 
+
+In Flink 1.17, daily performance monitoring has been integrated into the
+#flink-dev-benchmarks Slack channel. This feature is crucial in quickly
+identifying regressions and ensuring the quality of the code. Once a regression
+is identified through the Slack channel or the [speed
+center](http://codespeed.dak8s.net:8000), developers can refer to the guidance
+provided in the [Benchmark's
+wiki](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=115511847)
+to address the issue effectively. This feature helps the community take a
+proactive approach to ensuring system performance, resulting in a better product
+and increased user satisfaction.
+
+
+## Subtask Level Flame Graph
```

Comment:
Just as an idea: It might be nice to add a screenshot of that feature to the end of this section. :thinking:

Reply:
nice idea, will add a picture